site/tests/BaseUnitTest.php

```diff
@@ -9,8 +9,19 @@
 use app\models\Config;
 use app\models\User;
 use ReflectionException;
+use Doctrine\Common\Annotations\AnnotationRegistry;
+
 
 class BaseUnitTest extends \PHPUnit\Framework\TestCase {
+
+    /**
+     * Loads annotations for routers.
+     */
+    public static function setUpBeforeClass(): void {
+        $loader = require(__DIR__.'/../vendor/autoload.php');
+        AnnotationRegistry::registerLoader([$loader, 'loadClass']);
```

Comment:
Is this really needed for all unit tests or is it just needed for the WebRouter tester?

Reply:
It's only needed for WebRouter. Will move it.